dockerfiles/cpp-23.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gcc:13.2.0-bookworm
+
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="vcpkg.json,vcpkg-configuration.json"
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y zip=3.* && \ 
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN pwd && \
+    git clone https://github.com/microsoft/vcpkg.git && \
+    ./vcpkg/bootstrap-vcpkg.sh -disableMetrics
+
+RUN export "VCPKG_ROOT"="$(pwd)" && \
+    export PATH="$VCPKG_ROOT:$PATH"
+
+RUN echo "export PATH=/cmake-portable/bin:$PATH && cd \${CODECRAFTERS_SUBMISSION_DIR} && cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=/vcpkg/scripts/buildsystems/vcpkg.cmake" > /codecrafters-precompile.sh
```

Comment:
Can the PATH mangling be moved to a separate step above, instead of placing it inside precompile?